components/esp32/Kconfig

```diff
@@ -929,6 +929,24 @@ menu "ESP32-specific"
             This option depends on the CONFIG_FREERTOS_UNICORE option because RTC fast memory
             can be accessed only by PRO_CPU core.
 
+config USE_FIXED_STATIC_RAM_SIZE
+	bool "Use fixed static RAM size"
+	default false
```

Comment:
Minor, but Kconfig syntax should be `default n` (this will also work but our Kconfig linter will complain about it.)

Reply:
fixed.